resources/config.json

```diff
@@ -0,0 +1,13 @@
+{
+    "suites": [
+        {
+            "name": "NewsSite-PostMessage",
+            "url": "resources/newssite/news-next/dist/index.html",
+            "tags": ["default", "newssite", "language"],
```

Comment:
Why is this tag including "default"? Isn't it supposed to be "experimental" instead?